example/App.tsx

```diff
@@ -23,7 +23,11 @@ export default function App() {
         ) : (
           <View>
             <AuthView setClient={setClient} />
-            <Button onPress={() => setIsTesting(true)} title="Unit tests" />
+            <Button
+              onPress={() => setIsTesting(true)}
+              title="Unit tests"
+              accessibilityLabel="Unit-tests"
```

Comment:
accessibilityLabels need to be added so the android espresso tests can find things.